bitwarden_license/src/Commercial.Core/SecretsManager/AuthorizationHandlers/Secrets/SecretAuthorizationHandler.cs

```diff
@@ -81,6 +82,16 @@ public class SecretAuthorizationHandler : AuthorizationHandler<SecretOperationRe
         }
     }
 
+    private async Task<bool> checkAccessToSecret(Secret resource, Guid userId, AccessClientType accessClient)
```

Comment:
Let's move this below `CanUpdateSecretAsync` and use the C# convention of pascal casing on methods + async for async methods. 

`CheckAccessToSecretAsync`

It might be better to indicate we're fetching the access, not checking, something like `GetAccessToUpdateSecretAsync`

Reply:
Thanks Thomas! I am confused by the phrasing of "methods +async for async methods" am I missing something related to async methods in my method?

Reply:
No, let me try to clarify. 

My comment was around the conventions we use in the project for naming methods.

First is that for method names we want to have pascal casing. 

Second, for methods that use asynchronous code, we prefer to have the method end with `Async`. 

You can see this preference in the editor configuration file here https://github.com/bitwarden/server/blob/4c61d05b243fbdf8f230d1a2b6f3d2f88f53008e/.editorconfig#L62-L65